runtime/libpgmath/CMakeLists.txt

```diff
@@ -117,3 +122,11 @@ include(LibmathUtils)
 # Setup Source Code And Tests
 add_subdirectory(lib)
 add_subdirectory(test)
+
+if(0)
```

Comment:
Why an if(0)?

Reply:
The developer said it was left in for debug. I can have it removed.

Reply:
No need to do anything special.